src/main/views/manual-upload.njk

```diff
@@ -10,206 +10,231 @@
-                <div class="govuk-form-group">
-                  <label class="govuk-label govuk-date-input__label" for="display-date-to-month">
-                    {{ form.month }}
-                  </label>
-                  <input class="govuk-input govuk-date-input__input govuk-input--width-2 {{ displayToError }}" id="display-date-to-month" name="display-date-to-month" type="text" pattern="[0-9]*" inputmode="numeric"></div>
-              </div>
-              <div class="govuk-date-input__item">
-                <div class="govuk-form-group">
-                  <label class="govuk-label govuk-date-input__label" for="display-date-to-year">
-                    {{ form.year }}
-                  </label>
-                  <input class="govuk-input govuk-date-input__input govuk-input--width-4 {{ displayToError }}" id="display-date-to-year" name="display-date-to-year" type="text" pattern="[0-9]*" inputmode="numeric"></div>
+                <div class="govuk-date-input" id="display-date-to">
+                  <div class="govuk-date-input__item">
+                    <div class="govuk-form-group">
+                      <label class="govuk-label govuk-date-input__label" for="display-date-to-day">
+                        {{ form.day }}
+                      </label>
+                      <input class="govuk-input govuk-date-input__input govuk-input--width-2 {{ displayToError }}" id="display-date-to-day" name="display-date-to-day" type="text" pattern="[0-9]*" inputmode="numeric"></div>
+                  </div>
+                  <div class="govuk-date-input__item">
+                    <div class="govuk-form-group">
+                      <label class="govuk-label govuk-date-input__label" for="display-date-to-month">
+                        {{ form.month }}
+                      </label>
+                      <input class="govuk-input govuk-date-input__input govuk-input--width-2 {{ displayToError }}" id="display-date-to-month" name="display-date-to-month" type="text" pattern="[0-9]*" inputmode="numeric"></div>
+                  </div>
+                  <div class="govuk-date-input__item">
+                    <div class="govuk-form-group">
+                      <label class="govuk-label govuk-date-input__label" for="display-date-to-year">
+                        {{ form.year }}
+                      </label>
+                      <input class="govuk-input govuk-date-input__input govuk-input--width-4 {{ displayToError }}" id="display-date-to-year" name="display-date-to-year" type="text" pattern="[0-9]*" inputmode="numeric"></div>
+                  </div>
+                </div>
+              </fieldset>
               </div>
             </div>
-          </fieldset>
+          {{ submitButton() }}
+        </form>
+        {{ super() }}
+      </div>
+      <div class="govuk-grid-column-one-third">
+        <h2 class="govuk-heading-l">Page Help</h2>
 
-        </div>
+        <h3 class="govuk-heading-s">Hearing lists</h3>
```

Comment:
Rather than the raw text being in here, we should put the text in the resources/locales/en/manual-upload.json and reference in from there :)